src/components/Synthetics/StatusNotification/SubaccountNotification.tsx

```diff
@@ -0,0 +1,147 @@
+import { Trans, t } from "@lingui/macro";
+import cx from "classnames";
+import SpinningLoader from "components/Common/SpinningLoader";
+import ExternalLink from "components/ExternalLink/ExternalLink";
+import { getExplorerUrl } from "config/chains";
+import { useSubaccountNotificationState, useSubaccountPendingTx } from "context/SubaccountContext/SubaccountContext";
+import { useChainId } from "lib/chains";
+import { museNeverExist } from "lib/types";
+import { usePrevious } from "lib/usePrevious";
+import { ReactNode, memo, useMemo } from "react";
+import { useToastAutoClose } from "./useToastAutoClose";
+
+const SubaccountNotificationImpl = ({
+  toastId,
+  subaccountWasAlreadyGenerated,
+  subaccountWasAlreadyActivated,
+}: {
+  toastId: number;
+  subaccountWasAlreadyGenerated: boolean;
+  subaccountWasAlreadyActivated: boolean;
+}) => {
+  const hasError = false;
+
+  const [notificationState] = useSubaccountNotificationState();
+  const isCompleted = useMemo(() => {
+    return ["activated", "deactivated", "deactivationFailed", "activationFailed", "generationFailed"].includes(
```

Comment:
We can use an enum for `notificationState` values

Reply:
As I only use this type for internal state managing I don't see a reason for enums

Reply:
I agree it's not ok having array of strings here as our ts config wouldn't check it properly

I'll change it like so, ok?
<img width="582" alt="image" src="https://github.com/gmx-io/gmx-interface/assets/144131435/c2051ff7-decf-44ad-a1e2-410f05f79bbc">